src/cluster_legacy.c

```diff
@@ -46,6 +47,7 @@
 #include <sys/stat.h>
 #include <math.h>
 #include <sys/file.h>
+#include <stdbool.h>
```

Comment:
I'm going to be opinionated and say can we please not add bool now.

Reply:
I like bool as it makes the function contract a lot clearer. otherwise I will have to read either the code comment or the actual function implementation.

Reply:
This feels almost like a separate major decision since it's so deeply rooted in this code base. We have atomics and other C99 and later features, but not bool and C++ comments. :smile:

I'd like to hear arguments against adding it though. It's part of C nowdays and it's odd to have a rule that forbids it.

Reply:
sgtm. I will revert this change then. will open a separate issue to track that discussion.

Reply:
I'm not strictly against bool, but if we use it I would prefer we use it everywhere. I mentioned on the other thread we had an issue since someone changed the type of something from int to bool and it broke our clusterbus (it was really nasty).

Reply:
Discussion moved to #413.

Reply:
reverted `bool` changes